### PR TITLE
translate root breadcumbs

### PR DESCRIPTION
**Pull request by tamplan**
Hi,

Fix root breadcumbs element to be translated.

### CODE REVIEW

**Comment by tamplan** [comment]
Update french language files

File: resources/lang/fr/buttons.php [hunk 1]
```diff
@@ -18,14 +18,14 @@
             'users' => [
                 'activate'           => 'Activer',
                 'change_password'    => 'Changer de mot de passe',
-                'clear_session'         => 'Clear Session',
-                'confirm'             => 'Confirm',
+                'clear_session'       => 'Effacer la session',
```
Comment (bloggervista) [inline resources/lang/fr/buttons.php]: Alignment with other => I mean this line has different alignment than line 22, 21

Comment (tamplan) [inline resources/lang/fr/buttons.php]: Hi,
I do not feel still comfortable in use of git for the moment.
Did i need to do some changes?

Regards.

Comment (bloggervista) [inline resources/lang/fr/buttons.php]: Oh then no , not needed now. I will later change it :)

Comment (tamplan) [inline resources/lang/fr/buttons.php]: Ok, thanks you very much!

Comment (rappasoft) [inline resources/lang/fr/buttons.php]: Don't worry about formatting, StyleCI has an auto hook that formats when I pull to dev.

**Pull request merged**